packages/compass-welcome/src/components/welcome-image.tsx

```diff
@@ -15,7 +15,7 @@ export function WelcomeModalImage(props: SVGProps<SVGSVGElement>) {
         </clipPath>
         <style>
           {
-            '.g,.h{fill:none}.i{fill:#e3fcf7}.g,.h,.i,.j,.k,.l,.m,.o,.p{stroke:#000000;stroke-linecap:round;stroke-linejoin:round}.j{fill:#ffc010}.k,.o{fill:#ffffff}.l{fill:#ffc010}.m{fill:#00ed64}.h{stroke-dasharray:0 0 2 2}.o{stroke-width:.8px}'
```

Comment:
This is css, not css in js, so shouldn't be capilatized

Reply:
Good catch!